lib/fluent/plugin/filter_throttle.rb

```diff
@@ -4,7 +4,7 @@ module Fluent::Plugin
   class ThrottleFilter < Filter
     Fluent::Plugin.register_filter('throttle', self)
 
-    config_param :group_key, :string, :default => 'kubernetes.container_name'
+    config_param :group_key, :array, :default => ['kubernetes.container_name']
```

Comment:
maybe have group_key and group_keys and then combine them into group_keys to keep it backwards compatible ?

Reply:
group_key seems like a fine name. and the tests check that the array syntax is compatible with the single-case string syntax